### PR TITLE
Adding role arn explanation for sagemaker role

### PR DESCRIPTION
**Pull request by gautamkmr**
*Description of changes:*
Adding description about role-arn when job is being submitted using plain yaml. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by gautamkmr** [comment]
@RedbackThomson @cadedaniel

File: doc/amazon_sagemaker_operators_for_kubernetes_jobs.rst [hunk 1]
```diff
@@ -30,7 +30,12 @@ Download the sample YAML file for training using the following command:
     wget https://github.com/aws/amazon-sagemaker-operator-for-k8s/blob/master/samples/xgboost-mnist-trainingjob.yaml
 
 Edit the \ ``xgboost-mnist-trainingjob.yaml`` file to replace the ``roleArn`` parameter with your \ ``<sagemaker-execution-role>``, and \ ``outputPath`` with your S3 bucket that the Amazon SageMaker
-execution role has write access to. Apply the YAML file using the
+execution role has write access to. The ``roleArn`` must have permissions so that Amazon SageMaker
```
Comment (RedbackThomson) [inline doc/amazon_sagemaker_operators_for_kubernetes_jobs.rst]: Does this need a `\` before the \`\`roleArn\`\` ? I'm not really sure how rust documentation works.

**Comment by IvyBazan** [comment]
LGTM!

File: doc/amazon_sagemaker_operators_for_kubernetes_jobs.rst [hunk 2]
```diff
@@ -30,7 +30,12 @@ Download the sample YAML file for training using the following command:
     wget https://github.com/aws/amazon-sagemaker-operator-for-k8s/blob/master/samples/xgboost-mnist-trainingjob.yaml
 
 Edit the \ ``xgboost-mnist-trainingjob.yaml`` file to replace the ``roleArn`` parameter with your \ ``<sagemaker-execution-role>``, and \ ``outputPath`` with your S3 bucket that the Amazon SageMaker
-execution role has write access to. Apply the YAML file using the
+execution role has write access to. The \ ``roleArn`` must have permissions so that Amazon SageMaker
+can access Amazon S3, Amazon CloudWatch, and other services on your 
+behalf. For more information on creating an Amazon SageMaker
+ExecutionRole, see \ `Amazon SageMaker
```
Comment (laurenyu) [inline doc/amazon_sagemaker_operators_for_kubernetes_jobs.rst]: not sure why these backslashes were here before, but you don't need any of them

**Comment by gautamkmr** [comment]
Deleted the branch by mistake, reopening it [here](https://github.com/aws/sagemaker-python-sdk/pull/1219/files).

**Pull request closed without merging**